include/sys/zfs_context.h

```diff
@@ -401,6 +401,7 @@ void procfs_list_add(procfs_list_t *procfs_list, void *p);
 #define	KM_SLEEP		UMEM_NOFAIL
 #define	KM_PUSHPAGE		KM_SLEEP
 #define	KM_NOSLEEP		UMEM_DEFAULT
+#define	KM_ONCE			UMEM_DEFAULT
```

Comment:
Is this still used with the new pooling?

Reply:
same as before KM_ONCE is not in use anymore. but i'm thinking about using it by force to get it merged to zfs :-) since it a usefull feature. just kidding. i will remove it on next update